tests_config/run_spec.js

```diff
@@ -187,15 +188,23 @@ global.run_spec = (fixtures, parsers, options) => {
             codeForSnapshot = visualizeRange(codeForSnapshot, baseOptions);
             codeOffset = codeForSnapshot.match(/^>?\s+1 \| /)[0].length;
           }
+          let resultForSnapshot = formattedWithCursor;
 
           if (hasEndOfLine) {
+            // We snapshot the linux version
+            if (mainOptions.endOfLine === "auto" && os.EOL === "\r\n") {
```

Comment:
This part looks suspicious. `--end-of-line auto` doesn't mean "use `os.EOL`". It means 
> Maintain existing line endings (mixed values within one file are normalised by looking at what’s used after the first line)

Reply:
Thanks, I misunderstood.